docs/crio.conf.5.md

```diff
@@ -125,6 +125,9 @@ the container runtime configuration.
 **apparmor_profile**=""
   Used to change the name of the default AppArmor profile of CRI-O. The default profile name is "crio-default".
 
+**irqbalance_config_file**="/etc/sysconfig/irqbalance"
```

Comment:
can you add a blurb about where this file should be for which distro (only for the most popular: ubuntu/centos, maybe debian/rhel if they differ at all)

Reply:
Yes! done, I was able to confirm the file locations only in centos/suse and ubuntu platforms. so added it only for those cases.